source/server-discovery-and-monitoring/tests/README.md

```diff
@@ -71,6 +71,8 @@ following keys:
 
 - type: A ServerType name, like "RSSecondary". See [ServerType](../server-discovery-and-monitoring.md#servertype) for
     details pertaining to async and multi-threaded drivers.
+- error: An optional object with a with a string field containing a string that must be a substring of the message on
```

Comment:
`An optional object with a with a string field containing a string that must be...`
->
`An optional string that must be...`